services/QuillLMS/spec/lib/rule_feedback_history_spec.rb

```diff
@@ -20,7 +20,7 @@ def rule_factory(&hash_block)
       }.merge(yield)
     )
   end
-
+ 
```

Comment:
Accidental white space?

Reply:
No; we need one there. There wasn't one previously.